variables.tf

```diff
@@ -68,22 +68,20 @@ variable "s3_folder" {
   description = "The Amazon S3 bucket folder to use as path of the exported data"
 }
 
-variable "s3_lifecycle_configuration_rules" {
-  type = list(object({
-    enabled = bool
-    id      = string
-
-    abort_incomplete_multipart_upload_days = number
-
-    filter_and = any
-    expiration = any
-    transition = list(any)
-
-    noncurrent_version_expiration = any
-    noncurrent_version_transition = list(any)
-  }))
-  default     = []
-  description = "A list of lifecycle V2 rules"
+variable "s3_lifecycle_rules" {
+  type = object({
+    enabled                    = bool
+    expiration_days            = optional(number, 365)
+    glacier_transition_days    = optional(number, 0)
+    noncurrent_expiration_days = optional(number, 7)
+  })
+  default = {
+    enabled                    = true
+    expiration_days            = 1095
+    glacier_transition_days    = 365
+    noncurrent_expiration_days = 7
```

Comment:
for what?

Reply:
I'm not sure that I understand the question.

Reply:
I don't understand what's purpose of noncurrent rule here

Reply:
@SweetOps What current rules should be removed?
 Can you explain this to me?

Reply:
What to do with  purpose of noncurrent  rule?
@SweetOps Would you pls advise  me to delete or edit it?

Reply:
TODO:
* use the same variable for rules configuration as in original bucket [module](https://github.com/cloudposse/terraform-aws-s3-bucket/blob/1c227f1228c7286557d542088244deb8ce8356c8/variables.tf#L142C11-L142C40)  
* if you can explain why we should keep the `noncurrent_expiration_days` option, let's keep it because from my view the names of objects are uniq
* we need rules as default value: transition to glacier (n-days), expiration (n-days), delete multipart upload just in case and option to delete expired object markers. It could be one rule or several (AFAIR some options conflict with other )